faq/feeds_sdk/index.php

```diff
@@ -49,14 +49,12 @@
 files for Project Gutenberg's Distributed Proofreaders are created as 
 well as focusing mostly on how to implement these XML feeds into your 
 site.</p>
-<p>If you are further interested in XML you may want to look into the 
+<p>If you are further interested in XML and RSS you may want to look into the 
 follow sites as well:</p>
```

Comment:
"following" sites (I realize that was in the original)

Reply:
Good catch - fixed.